README.md

```diff
@@ -56,6 +56,20 @@ All html patterns in your pattern library will be located in `./patterns`. They
 
 Any changes to files in the `./patterns` directory will automatically be imported into your local Pattern Lab instance as long as you have run the `gulp serve` command and your server is running.
 
+## Can I include one pattern in another?
+
+as a twig include, yes.
+
+ * [how to twig includes](http://twig.sensiolabs.org/doc/tags/include.html)
+ * [example pattern with include](https://github.com/pattern-library/pattern-library/blob/master/patterns/molecules/media/figure-image/figure-image.twig)
+
+## Can I modify an existing pattern?
```

Comment:
i mean the promo pattern (since that was already there for example) or another NPM-based existing pattern which you're saying we can pull in right? does that make sense?